deploy/helm-chart/templates/rbac.yaml

```diff
@@ -39,8 +39,7 @@ rules:
   - get
   - list
   - watch
-# FIXME(bjee19): make nodes, pods, replicasets permission dependent on telemetry being enabled.
-# https://github.com/nginxinc/nginx-gateway-fabric/issues/1317.
+{{- if or .Values.nginxGateway.productTelemetry.enable .Values.nginx.plus }}
```

Comment:
the usage report and product telemetry ones do not fully intersect.

For example,
usage report doesn't need to get pods.

At  the same time, telemetry doesn't need to list replica set

Could we separate those permissions?